plugins/hydra_rq_launcher/setup.py

```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-rq-launcher",
-        version="1.0.1",
```

Comment:
I already bumped the version.
1. Undo the manual bump.
2. I land
3. publish
4. Optionally bump the repo version with the command mentioned in #944.

Reply:
Okay, I fixed the version.

Reply:
Great, are you going to publish to PyPI or should I?

Reply:
go ahead.

Reply:
👍 released it